common-graphql/src/main/resources/graphql/schemas/ITC.graphql

```diff
@@ -4718,9 +4746,32 @@ type Result {
 }
 
 type SpectroscopyResult {
-  serverVersion: NonEmptyString!
+
+  serverVersion: String!
 
   # Search results
   results: [Result!]!
 }
 
+type ItcSeries {
+  title: String!
+
+  # Data type: alternatives SignalData, BackgroundData, SingleS2NData, FinalS2NData, PixSigData, PixBackData
```

Comment:
Can this be an `enum`?

Reply:
as in graphql enum?
I need to go back to the schema and will review that